js/src/redux/providers/signerMiddleware.js

```diff
@@ -120,7 +120,7 @@ export default class SignerMiddleware {
         });
     }
 
-    handlePromise(this._api.signer.confirmRequest(id, { gas, gasPrice }, password));
+    handlePromise(this._api.signer.confirmRequest(id, { gas, gasPrice }, password.password));
```

Comment:
the weirdness of calling an object  with `password` and `wallet` fields `password` was already there. if this is odd, i suggest that variable be renamed rather than adding superlative logic here.